src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -47,25 +49,38 @@
 
     public LogzioSender(String logzioToken, String logzioType, int drainTimeout, int fsPercentThreshold, String bufferDir,
                         String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
-                        LogzioLogbackAppender.StatusReporter reporter, ScheduledExecutorService tasksExecutor) throws IllegalArgumentException {
+                        LogzioLogbackAppender.StatusReporter reporter, ScheduledExecutorService tasksExecutor, boolean addHostname, String additionalFields) throws IllegalArgumentException {
```

Comment:
I like to be explicit: Optional<String> additionalFields.
I would also JavaDoc the format above the c'tor